app.js

```diff
@@ -1,9 +1,13 @@
-const express = require('express');
-const { getAllTopics } = require('./controllers/topics.controllers');
-
+const express = require("express");
+const { getAllTopics } = require("./controllers/topics.controllers");
+const endpoints = require("./endpoints.json");
 const app = express();
 
-app.get('/api/topics', getAllTopics);
+app.get("/api", (req, res) => {
+  res.status(200).send(endpoints);
```

Comment:
Line 7, remember to .send an object on a new key. For example, `.send( { endpoints } )` then access the response data on the key of `endpoints` in the test

Reply:
done